### PR TITLE
Improvements to gradient editor at small size

### PR DESCRIPTION
**Pull request by runrevben**

### CODE REVIEW

File: extensions/widgets/gradientrampeditor/gradientrampeditor.lcb [hunk 1]
```diff
@@ -256,9 +282,9 @@ public handler OnPaint() returns nothing
       clip to mGradientRectangle on this canvas
       fill this canvas
    end layer on this canvas
+end handler
 
-   -- Draw the gradient on top of the checkerboard.
-   --
+private handler onParintGradient() returns nothing
```
Comment (livecodeali) [inline extensions/widgets/gradientrampeditor/gradientrampeditor.lcb]: Little typo here

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 3e17cf6

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 3e17cf6f2117ec698acbe5f33ec45efc0b413c60

**Comment by livecode-vulcan** [comment]
😎 test success 3e17cf6f2117ec698acbe5f33ec45efc0b413c60
- try-linux-x86: [success](https://vulcan.livecode.com/builders/try-linux-x86/builds/581)
- try-win-x86: [success](https://vulcan.livecode.com/builders/try-win-x86/builds/641)
- try-ios: [success](https://vulcan.livecode.com/builders/try-ios/builds/530)
- try-android-armv6: [success](https://vulcan.livecode.com/builders/try-android-armv6/builds/594)
- try-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-linux-x86_64/builds/526)
- try-mac: [success](https://vulcan.livecode.com/builders/try-mac/builds/519)

**Pull request merged**